emperor/support_files/js/view.js

```diff
@@ -106,112 +106,36 @@ DecompositionView.prototype.changeVisibleDimensions = function(newDims){
 };
 
 /**
- *
  * Change the plottables coloring based on the metadata category using the
  * provided color function
  *
- * @param {colorFunc} a Function that accepts a category value and returns
- * a color
- * @param {category} a string with the metadata header.
- * @return {dataview} Array of objects to be consumed by Slick grid.
+ * @param {attributes} key:value pares of elements and values to change in
+ * plottables
+ * @param {setPlottableAttributes} helper function to change the values of
+ * plottables, in general this should be implemented in the controller but it
+ * can be nullable if not needed
```

Comment:
it would be helpful to briefly describe the arguments that are passed to this function, and perhaps point to a concrete example i.e. `see ColorViewController.setPlottableAttribute ....` or something like that.